go/state/state_db.go

```diff
@@ -823,6 +829,34 @@ func (s *stateDB) SetState(addr common.Address, key common.Key, value common.Val
 	}
 }
 
+func (s *stateDB) GetTransientState(addr common.Address, key common.Key) common.Value {
+	sid := slotId{addr, key}
+	val, _ := s.transientStorage.Get(sid)
+	return val
+}
+
+func (s *stateDB) SetTransientState(addr common.Address, key common.Key, value common.Value) {
+	sid := slotId{addr, key}
+	if entry, exists := s.transientStorage.Get(sid); exists {
```

Comment:
`entry` -> `value`

in Maps, `entry` usually refers to the pair of `key-value`, but here you get the `value`, not the whole entry

Reply:
I used the same naming as in [`SetState`](https://github.com/Fantom-foundation/Carmen/blob/dddf65b4e1f3b09c884d10eb97d23399cf07ea74/go/state/state_db.go#L794) but I guess it make sense to use value here more.